packages/mobile/src/screens/notifications-screen/NotificationsScreen.tsx

```diff
@@ -31,6 +31,7 @@ export const NotificationsScreen = () => {
         text={messages.header}
         icon={IconNotification}
         iconProps={{ height: 28, width: 28 }}
+        styles={{ icon: { marginLeft: -1 } }}
```

Comment:
how come this is necessary?

Reply:
Yeah it's weird, the icons have various widths, so this is an attempt to align each icon